### PR TITLE
OSDOCS10283: Added a known issue regarding oc-mirror.

### PR DESCRIPTION
**Pull request by subhtk**
Version(s): 4.15

Issue: [OSDOCS-10283](https://issues.redhat.com/browse/OSDOCS-10283)

Link to docs preview: [Search with 31536](https://75077--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-15-release-notes.html#ocp-4-15-known-issues)

SME review:
- [x] SME has approved this change.

QE review:
- [x] QE has approved this change.

Additional information: [OCPBUGS 31536](https://issues.redhat.com/browse/OCPBUGS-31536)

### CODE REVIEW

File: release_notes/ocp-4-15-release-notes.adoc [hunk 1]
```diff
@@ -2662,6 +2662,8 @@ A proposed fix is to report a `FREERUN` event when the NMEA string is lost.
 
 * If you have IPsec enabled on the cluster, on the node hosting the north-south IPsec connection, restarting the `ipsec.service` systemd unit or restarting the `ovn-ipsec-host` pod causes a loss of the IPsec connection. (link:https://issues.redhat.com/browse/RHEL-26878[*RHEL-26878*])
 
+* When mirroring operator catalogs, `oc-mirror` rebuilds the catalogs and regenerates their internal cache according to `imagesetconfig` catalog filtering specifications. This operation requires the use of the `opm` binary found within the catalogs. In 4.15, operator catalogs include the `opm` RHEL 9 binary, causing the mirroring process to fail when running on RHEL 8 systems. (link:https://issues.redhat.com/browse/OCPBUGS-31536[*OCPBUGS-31536*])
```
Comment (agantony) [inline release_notes/ocp-4-15-release-notes.adoc:2665]:
```suggestion
* Currently, there is a known issue with the mirroring of operator catalogs. The `oc-mirror` rebuilds the catalogs and regenerates their internal cache according to the `imagesetconfig` catalog filtering specifications. This operation requires the use of the `opm` binary contained in the catalogs. In OpenShift Container Platform 4.15, the operator catalogs contain the `opm` RHEL 9 binary, which causes the mirroring process to fail on RHEL 8 systems. (link:https://issues.redhat.com/browse/OCPBUGS-31536[*OCPBUGS-31536*])
```

**Review by agantony — COMMENTED** [review]
Added a suggestion for your consideration; otherwise lgtm!

File: release_notes/ocp-4-15-release-notes.adoc (hunk 1 above)
Comment (subhtk) [reply release_notes/ocp-4-15-release-notes.adoc:2665]: Ack.

**Pull request merged**